### ISSUE TITLE
redesign Download button

### ISSUE THREAD

**Issue by shcheklein** [iterative/dvc.org#919]
We need yet another iteration. I think we should rename it (`Install` instead of `Download`) and make a link to the `Install` section in docs. @efiop @jorgeorpinel @casperdcl thoughts? 

Otherwise we are forcing people installing "binary" py-installer version. Moreover it does not take into account the new need to describe `dvc as a library` installation method, etc. It's just not scalable anymore.

**Comment by jorgeorpinel** [comment]
I agree. Downloading the installer is only the recommended method for Windows and even then "Install" is a better legend for the button. It already has a ink to /doc/install though, in the smaller, top option "pip, conda, brew". So is the change just "Download" -> "Install"?

> new need to describe `dvc` as a library

Good point I'll add some notes in the install section along with #908.

**Comment by shcheklein** [comment]
> So is the change just "Download" -> "Install"?

yes, pretty much. Also, remove that dropdown completely and simplify code around.

**Comment by jorgeorpinel** [comment]
> Good point I'll add some notes in the install section 

Done in 4a50af91

> remove that dropdown completely and simplify code around

OK. We could just keep the platform detection so it links to the exact install page 🙂

**Comment by shcheklein** [comment]
> OK. We could just keep the platform detection so it links to the exact install page 🙂

 Yes! good idea

**Comment by efiop** [comment]
More people are complaining https://github.com/iterative/dvc/issues/5192 . Could we prioritize this maybe? We'll consider dropping binary packages for 2.0 in general soon https://github.com/iterative/dvc/issues/4841 , so this might become even more relevant.

**Comment by julieg18** [comment]
Are we still wanting to do this? cc @iterative/docs @iterative/websites
